directus/Caddyfile

```diff
@@ -0,0 +1,28 @@
+# change to the hostname you want to bind to
+localhost:3000
+
+# change to correct document root
+# in this example config directus is assumed to be root
+# you may need to adjust rewrites according to your own setup
+root /var/www/directus
+
+# Set the path to the php-fpm process.
```

Comment:
Oh, okay. In that case, maybe rephrase this; the path doesn't point to the php-fpm process.